devops/vaults.json

```diff
@@ -57,5 +57,11 @@
             "base",
             "business-create-ui"
         ]
+    },
+    {
+        "vault": "hotjar",
+        "application": [
+            "shared"
+        ]
```

Comment:
This is a new vault in 1Password. The "shared" application (section) contains keys that are shared between projects. There are other sections specific to each project.

Example (key ids are not yet updated):

![image](https://user-images.githubusercontent.com/10002889/214180794-d5b940fe-9660-492f-ab8d-9e94c64f7a6b.png)